app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+  before_action :configure_account_update_params, only: [:update]
+
+  # GET /resource/sign_up
+  def new
+    super
+  end
+
+  #POST /resource
+  def create
+    super do |resource|
+      if resource.valid?
+        @organization = Organization.create(name: params[:organization_name])      
```

Comment:
what will happen if organization_name is blank?

Reply:
use permitted_params to sanitize request data